src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -14,11 +14,20 @@ import {
 } from '@material-ui/core';
 import { orderBy } from 'lodash';
 import { useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
 import { TableRow as AnalysisTableRow } from '../../../../context/analysisResultStateSlice';
 import { showPopup } from '../../../../context/tooltipStateSlice';
 import { Column } from '../../../../utils/analysis-utils';
+import { formattedTranslation } from '../../../../i18n';
 
 function AnalysisTable({ classes, tableData, columns }: AnalysisTableProps) {
+  // only display local names if local language is selected, otherwise display english name
+  const { i18n } = useTranslation();
+  const filteredColumns = columns.filter(column =>
```

Comment:
I think a better way to do this would be to just have a "name" column and change the value inside.

Note that we also need to translate the column names

Reply:
And for now you can simply use `i18n.resolvedLanguage === "en"`

Reply:
Good idea, done